### DISCUSSION TITLE
Apache2 Server Monitoring Support

### DISCUSSION THREAD

**Discussion by AjinThankachan** [apache/skywalking#9155] [Q&A] [answered]
Hi, 
Using Skywalking is it possible to monitor the Apache2 service. The mod info is already enabled so using it or using any agent is it possible to do the monitoring. I was testing an application which is having apache2 server standalone and the application is in another vm. So i have configured the application but apache server i am unable to get data and unable to monitor. Is it possible to do the monitoring? If not is there any possibility that we can add it. Thanks in advance. 

Thanks

**Comment by wu-sheng** [comment, marked as answer]
I don't think we have apache2 agent, AFAIK.
If Apache2 has Prometheus endpoint, or OpenTelemetry supported metrics you could ship metrics to skywalking, as well as logs.

**Reply by AjinThankachan** [reply]
Thanks. We have created one endpoint to display data which we got using the mod in apache. So can we pass it to skywalking. Attaching a screenshot of the same. Can it be done via promptus or open telemetry? Else to populate the data what is the solution or approach that we can use. 
![image](https://user-images.githubusercontent.com/25667606/171327159-4b36a8d1-fc69-48a2-b9ce-0a78217129db.png)

**Reply by wu-sheng** [reply]
You could use OpenCensus format through OpenTelemetry collector, or skywalking native meter format as you like.

**Reply by AjinThankachan** [reply]
Thanks a lot for your support
